src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java

```diff
@@ -56,6 +56,8 @@ public AddTaskCommand parse(String args) throws ParseException {
         Link link = ParserUtil.parseLink(argMultimap.getValue(PREFIX_LINK));
 
         try {
+            dateTimeFormatter.setLenient(false);
```

Comment:
Just checking, will this prevent the input 32-13-2022 2539?

Reply:
Yeap as shown below
<img width="739" alt="Screenshot 2022-04-05 at 7 31 34 PM" src="https://user-images.githubusercontent.com/22110871/161744787-c73f09ed-9414-4618-a4dd-527544e5fdbb.png">